playground/README.md

```diff
@@ -0,0 +1,82 @@
+# CoW Playground 
+
+The ultimate goal is to have a single command that will spin up a local CoW network, with all the components needed to run the CoW Protocol stack 🚀
+
+## Quickstart
+
+1. Clone this repo.
+2. Change directory to `playground`.
+3. Configure the stack by editing the `.env.example` file and renaming it to `.env`. **NOTE**: RPC demand is very high, for optimal performance, use a local node. The stack was tested with `reth` on `mainnet`.
+4. Run `docker-compose -f docker-compose.fork.yml up -d`.
+5. Configure Rabby Wallet to use the RPC endpoint at `http://localhost:8545` (for `mainnet`, or your network of choice).
+6. Configure Rabby Wallet to use a test account (any of the first 10 accounts from the test mnemonic will do).
+
```

Comment:
Something nice to write is how to clear up all state that has been accumulated by Docker so you can try out a new network, since I believe by default the database data is kept. Just a mention to `docker-compose -f docker-compose.fork.yml down --remove-orphans --volumes` would probably be enough.

Reply:
Suggestions are happily merged :grin: